Godeps

```diff
@@ -17,8 +17,10 @@ github.com/peterh/liner 88609521dc4b6c858fd4c98b628147da928ce4ac
 github.com/philhofer/fwd 1612a298117663d7bc9a760ae20d383413859798
 github.com/retailnext/hllpp 38a7bb71b483e855d35010808143beaf05b67f9d
 github.com/spaolacci/murmur3 0d12bf811670bf6a1a63828dfbd003eded177fce
+github.com/stretchr/testify 890a5c3458b43e6104ff5da8dfa139d013d77544
```

Comment:
I agree with not using other _testing_ packages. This package is included only to leverage the assertion package `testify/assert`. It removes a lot of noise from unit tests when asserting for expected behavior / values. If the assertion fails, it will generate a standard t.Error, t.Fail, etc. For more complex types, like collections, diffs will also be produced. 

`testify/assert` contains APIs for simplifying a number of scenarios and uses the standard Go `*testing.T` type to report errors or failures. Some example

* [assert.PanicsWithValue](https://github.com/influxdata/influxdb/pull/8947/files#diff-82e168f130315192395e34c360d6910eR18) – asserts the code panics and returns an expected value. Much simpler than writing `defer`s, etc

Others I like include:

* `assert.NotNil`
* `assert.NoError`
* `assert.Empty`

This little [introduction](https://github.com/stretchr/testify#assert-package) is helpful. You can also review [the docs](https://godoc.org/github.com/stretchr/testify/assert) for the available assertions.

Reply:
I'm aware of this testing package. But, we don't use it anywhere else and I don't like the idea of including an additional dependency for only one section of code. Introducing this likely means that it will start being used in other locations and the unit tests won't have the same pattern anymore. They don't currently, but adding additional unit test patterns to a code base that already has a bunch seems like too much to me.

Reply:
Removed in favor of `pkg/testing/assert`